modules/nw-about-multicast.adoc

```diff
@@ -1,6 +1,16 @@
 // Module included in the following assemblies:
 //
-// * networking/using-multicast.adoc
+// * networking/openshift-sdn/using-multicast.adoc
+// * networking/ovn-sdn/enabling-multicast.adoc
+
+ifeval::["{context}" == "using-multicast"]
+:openshift-sdn:
+:sdn: OpenShift SDN
+endif::[]
+ifeval::["{context}" == "ovn-sdn-enabling-multicast"]
+:ovn:
```

Comment:
Do you actually use this? I see it defined/undefined but don't see it used.

Reply:
@bergerhoffer, right now it just sets the `sdn` name. I wanted to leave the door open for also having specific content for ovn-kube if it offers any particular behavior that doesn't apply to OpenShift SDN.

Reply:
@bergerhoffer, looks like I mixed some of my context, though. I'll fix them.